docs/content/preview/reference/drivers/java/postgres-jdbc-reference.md

```diff
@@ -32,15 +32,19 @@ type: docs
 
 </ul>
 
-The [PostgreSQL JDBC driver](https://jdbc.postgresql.org/) is the official JDBC driver for PostgreSQL, and can be used for connecting to YugabyteDB YSQL. YugabyteDB YSQL has full compatibility with the PostgreSQL JDBC Driver, allowing Java programmers to connect to YugabyteDB database to execute DMLs and DDLs using the JDBC APIs.
+The [PostgreSQL JDBC driver](https://jdbc.postgresql.org/) is the official JDBC driver for PostgreSQL, and can be used for connecting to YugabyteDB YSQL. YugabyteDB YSQL has full compatibility with the PostgreSQL JDBC Driver, allowing Java programmers to connect to YugabyteDB database to execute DMLs and DDLs using the JDBC APIs. The driver supports the [SCRAM-SHA-256 authentication method](../../../../secure/authentication/password-authentication/#scram-sha-256).
+
+For building a sample Java application with the PostgreSQL JDBC driver, see [Connect an application](../../../../drivers-orms/java/postgres-jdbc/).
 
 ## Download the driver dependency
 
+To download binary JAR files, go to [PostgreSQL JDBC driver – Downloads](https://jdbc.postgresql.org/download/). Because Java is platform neutral, download the appropriate JAR file and drop it into the classpath.
+
 PostgreSQL JDBC Driver is available as a maven dependency, and you can download the driver by adding the following dependency into the Java project.
 
 ### Maven dependency
 
-To get the driver from Maven, add the following dependencies to the Maven project:
+To get the driver from [Maven](https://maven.apache.org), add the following dependencies to the Maven project:
 
 ```xml
```

Comment:
The version to add the dependency is "42.2.14" , but the latest one as mentioned [here](https://deploy-preview-19562--infallible-bardeen-164bc9.netlify.app/preview/drivers-orms/java/#supported-projects) is "42.3.4". I think the dependency versions need to be updated on the reference pages as well.